index.html

```diff
@@ -1,17 +1,82 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width">
-    <title>repl.it</title>
-    <link href="style.css" rel="stylesheet" type="text/css" />
-  </head>
-  <body>
-    <div class='madLibsEdit'>
+
+<head>
+
+  <meta charset="utf-8">
+  <meta name="viewport"
+    content="width=device-width">
+  <title>The Cosmic Party | By Fennecs Team</title>
+  <link rel="icon"
+    type="image/x-icon"
+    href="./assets/sun.png"> <!--added a favicon from assets-->
+  <link href="style.css"
+    rel="stylesheet"
+    type="text/css" />
+  <link href="https://fonts.googleapis.com/css2?family=Bangers&display=swap"
```

Comment:
It's generally better to add Google Fonts to your CSS file rather than directly in your HTML file.